script/tool/test/drive_examples_command_test.dart

```diff
@@ -1023,6 +1023,7 @@ void main() {
                   'test',
                   '-d',
                   _fakeAndroidDevice,
+                  '--no-enable-impeller',
```

Comment:
Can you add a comment to remove this when https://github.com/flutter/flutter/issues/152691 is resolved?

Reply:
The comment is on the code; the test will fail once that is fixed. I'm also planning to file a separate issue just tracking the removal of the line to make very sure we don't forget.

Reply:
Facepalm